torch/lib/c10d/Utils.hpp

```diff
@@ -620,6 +660,24 @@ inline std::string recvString(int socket) {
   return std::string(value.data(), value.size());
 }
 
+inline void closeSocket(int socket) {
+#ifdef _WIN32
+  closesocket(socket);
```

Comment:
I'd also recommend using `::closesocket`. And add the `::` prefix in similar places. It should work on VC++.

Reply:
Resolved